src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -414,6 +335,18 @@ s! {
         __pos: off64_t,
         __state: crate::mbstate_t,
     }
+
+    // linux x32 compatibility
+    // See https://sourceware.org/bugzilla/show_bug.cgi?id=16437
+    pub struct timespec {
+        pub tv_sec: time_t,
+        #[cfg(all(gnu_time_bits64, target_endian = "big"))]
+        __pad: i32,
+        #[cfg(not(all(target_arch = "x86_64", target_pointer_width = "32")))]
+        pub tv_nsec: c_long,
```

Comment:
@snogge it seems like this config made the field disappear on 32-bit, https://github.com/rust-lang/rust/pull/142535#issuecomment-2974247689. Any chance you're able to put a fix up soon? If not, I'll post something today.